ios/app/app/MainBaseViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
```

Comment:
I think, we should let the candidate create the tableview and pull the data and show it.